intranet/static/themes/snow/snow.js

```diff
@@ -5,426 +5,427 @@
 
 var snowroot = "/static/themes/snow/";
 
-if(navigator.appName == 'Microsoft Internet Explorer') {
-	var ie=true;
-} else {
-	var ie=false;
-}
-var mobile=false;
-if(navigator.userAgent.toLowerCase().indexOf("android") != -1) {
-	var android=true;
-	var mobile=true;
-} else {
-	var android=false;
-}
-var fastbrowser = false;
-if(!ie) fastbrowser = true;
+var ie = (navigator.appName === 'Microsoft Internet Explorer');
+
+var android = (navigator.userAgent.toLowerCase().indexOf("android") !== -1);
+var mobile = android;
+
+var fastbrowser = !ie;
+
 //Config
 //Number of flakes
 
-if(typeof snowmax != 'undefined') {
-
-} else if(mobile) {
-	var snowmax = 15;
-} else {
-	var snowmax = 100;
+if (typeof snowmax === 'undefined') {
+    var snowmax = mobile ? 15 : 100;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['snowmax' was used before it was defined.](https://www.codacy.com/app/tjcsl/ion/file/2746716172/issues/source?bid=3176160&fileBranchId=3355598#l19)